spec/kahuna_client/client/adaptive_campaign_spec.rb

```diff
@@ -0,0 +1,47 @@
+require File.expand_path('../../../spec_helper', __FILE__)
```

Comment:
Yeah I don't think they're right (or this is using a pretty old version of rspec)